parser-typechecker/src/Unison/Runtime/Rt1IO.hs

```diff
@@ -461,7 +461,7 @@ runtime = Runtime terminate eval (nullaryMain External) (nullaryTest External) T
   terminate :: IO ()
   terminate = pure ()
   eval cl' ppe term = do
-    let cl = void (hoist (pure . runIdentity) IOSrc.codeLookup) <> cl'
+    let cl = cl' <> void (hoist (pure . runIdentity) IOSrc.codeLookup)
```

Comment:
Swapping the order of these code lookups just ensured that `IOSrc` isn't loaded until it's needed. This code is going away soon anyway when we get rid of the old runtime.